core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/JbrResolvers.scala

```diff
@@ -11,19 +11,22 @@ import org.virtuslab.ideprobe.dependencies.Dependency.Missing
 import org.virtuslab.ideprobe.Extensions.PathExtension
 
 object JbrResolvers {
-  val official =
-    JbrPatternResolver(
-      "https://cache-redirector.jetbrains.com/intellij-jbr/jbr_dcevm-[major]-[platform]-x64-b[minor].tar.gz"
-    )
+  // The `jbr_dcevm` pattern applies only to versions of IntelliJ older than 2022.2 release, whereas
+  // `jbr` pattern applies to newer versions where DCEVM is bundled by default.
+  private val officialJbrVersionPatterns = Seq(
+    "https://cache-redirector.jetbrains.com/intellij-jbr/jbr_dcevm-[major]-[platform]-[architecture]-b[minor].tar.gz",
+    "https://cache-redirector.jetbrains.com/intellij-jbr/jbr-[major]-[platform]-[architecture]-b[minor].tar.gz"
```

Comment:
Hmmm where is gonna the value for `architecture` be provided from now? 🤔

Reply:
As `JbrPatternResolver#resolve` doesn't know about that anymore

Reply:
OK, forgot to switch back from `[architecture]` to `x64` - let me fix

Reply:
done 👍